tests/test_utils.py

```diff
@@ -375,3 +375,13 @@ def test_multiline_redaction(self):
             {},
             msg="Got redactions on a multiline redaction, but shouldn't have.",
         )
+
+    def test_crosshatch_bad_redaction(self):
+        """Test gray scaled [X][X][X] redactions"""
+        path = root_path / "bad_cross_hatched_redactions.pdf"
+        bad_redactions = xray.inspect(path)
+        self.assertNotEqual(
+            bad_redactions,
+            {},
+            msg="Missing cross hatch bad redactions.",
+        )
```

Comment:
I think this might be better if the assertion checked for what we *did* want, so that the test can show what's expected. For example, above we do:

```
self.assertEqual(len(redactions[1]), 3)
```

Better to say: "I assert this document has X bad redactions" instead of "I assert this document does not have zero bad redactions."